src/js/containers/aboutTheData/AgenciesContainer.jsx

```diff
@@ -261,27 +261,31 @@ const AgenciesContainer = ({
         }) => [
             (<DrilldownCell data={agencyName} id={code} searchTerm={searchTerm} />),
             (<div className="generic-cell-content">{percentageOfTotalFederalBudget}</div>),
-            (<CellWithModal
-                data={mostRecentPublicationDate}
-                openModal={openModal}
-                modalType="publicationDates"
-                agencyData={{
-                    agencyName,
-                    agencyCode: code,
-                    fiscalYear: selectedFy,
-                    fiscalPeriod: selectedPeriod?.id
-                }} />),
-            (<CellWithModal
-                data={GtasNotInFileA}
-                openModal={openModal}
-                modalType="missingAccountBalance"
-                agencyData={{
-                    agencyName,
-                    gtasObligationTotal: _gtasObligationTotal,
-                    agencyCode: code,
-                    fiscalYear: selectedFy,
-                    fiscalPeriod: selectedPeriod?.id
-                }} />),
+            (mostRecentPublicationDate === '--' ?
```

Comment:
My only concern is that if the "null" API response is ever changed, this will silently fall back to always showing the expansion icon.

Reply:
Good point, let me see if I can hook into the private/raw property from our data model instead